game_of_life_test_review.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import game_of_life
+import test
+
+
+def test_newCellStateTrue():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) == "."
+
+
+def test_newCellStateFalse():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) != "x"
```

Comment:
You have so many corner cases (9 if branches) and only 2 tests for them - feels like not enough